Linting failed with 

```
 		pkg/cmd/upgrade/upgrade_cli.go:53:1: File is not properly formatted (gofmt)
			BinaryDownloadBaseURL  = "https://github.com/jenkins-x/jx/releases/download/v"
```

--- a/pkg/cmd/upgrade/upgrade_cli.go
+++ b/pkg/cmd/upgrade/upgrade_cli.go
@@ -50,7 +50,7 @@ var (
 
 const (
 	// BinaryDownloadBaseURL the base URL for downloading the binary from - will always have "VERSION/jx-OS-ARCH.EXTENSION" appended to it when used
-	BinaryDownloadBaseURL  = "https://github.com/jenkins-x/jx/releases/download/v"
+	BinaryDownloadBaseURL = "https://github.com/jenkins-x/jx/releases/download/v"
 	// LatestVersionstreamURL default version stream
 	LatestVersionstreamURL = "https://github.com/jenkins-x/jx3-versions.git"
 )

